src/app/Library/CrudPanel/Traits/HasViewNamespaces.php

```diff
@@ -4,6 +4,9 @@
 
 use Backpack\CRUD\ViewNamespaces;
 
+/**
+ * @codeCoverageIgnore
+ */
```

Comment:
Hmm... why ignore these? Aren't they actually pretty important?

Reply:
Not used, they are there just as a placeholder for broken compatibility with CRUD and PRO in some version range. (Explained in the doc block). 

What we use are in the `ViewNamespaces` class, that I added to tests and tested. 

Cheers

Reply:
Ah ok I understand. It still makes sense to test those, to make sure we don't break that backwards-compatibility, right? Might not, if we remove them in the next version (🤷‍♂️ why would we) but I'm just saying. Ignoring code coverage for a public method seems fishy to me.

Reply:
I think we should remove this from our codebase in the next version, when next major PRO requires next major CRUD this code shouldn't be there. 
This is a workaround for people that PRO license expired in a specific version. I get why we added this workaround at that time, but it's time for us to move on and assume that backpack pro 1.x is not compatible with crud v6.